vision/cloud-client/detect/beta_snippets_test.py

```diff
@@ -46,9 +47,25 @@ def test_handwritten_ocr(capsys):
 
 
 def test_handwritten_ocr_uri(capsys):
-    uri = 'gs://cloud-samples-data/vision/handwritten.jpg'
+    uri = GCS_ROOT + 'handwritten.jpg'
 
     beta_snippets.detect_handwritten_ocr_uri(uri)
 
     out, _ = capsys.readouterr()
     assert 'Cloud Vision API' in out
+
+
+def test_detect_pdf_document(capsys):
+    file_name = os.path.join(RESOURCES, 'kafka.pdf')
```

Comment:
test also TIFF and GIF.

Reply:
That means we are testing the actual API which is beyond the scope of this test.

Reply:
I thought we usually try to also test/validate the API. 

We get back the result and verify it's what we expected the API to return.

Reply:
> I thought we usually try to also test/validate the API.
> We get back the result and verify it's what we expected the API to return.

+1 yep

As for whether to test multiple file types, I leave that up to your discretion. I consider that "above and beyond." I wouldn't complain if we had that test. But I also don't think it's necessary.

Precedents:
 - We do not test all of the various types of audio formats which Speech accepts, e.g. mp3/wav/raw/...
 - We do not test all of the various image formats which Vision accepts, e.g. png/jpg/...